### DISCUSSION TITLE
Add WebDAV Support for NextCloud Integration ?

### DISCUSSION THREAD

**Discussion by LittleDuke** [opencapco/opencap.co#384] [Ideas]
We use Nextcloud for our dataroom system and have already integrated WebDAV in our investment crowdfunding portal to push Signed Subscription agreements.

Would be super useful (and likely valuable) to add in support for WebDAV.

Thoughts on Level of Effort ?

**Comment by dahal** [comment]
> We use Nextcloud for our dataroom system and have already integrated WebDAV in our investment crowdfunding portal to push Signed Subscription agreements.
> 
> Would be super useful (and likely valuable) to add in support for WebDAV.
> 
> Thoughts on Level of Effort ?

@LittleDuke Our current data room, document storage and document signing features works with any S3 compatible file storage (eg: Minio (open-source), AWS S3, Cloudflare R2, Google cloud storage), and I believe [NextCloud also supports S3 compatible storage](https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/primary_storage.html). We currently do not have a plan to support WebDAV (yet). We can look into this and see if it makes sense to prioritize. Another possible way to sync file could be using API as well.

**Reply by LittleDuke** [reply]
We host our instance on prem so an S3 storage would have zero value to us.   Put it in the backlog for scoping and come up with a level of effort and we might be willing to support it in some way (either funding or coding)
